Buildings/Applications/DHC/CentralPlants/Cooling/Subsystems/CoolingTowerWithBypass.mo

```diff
@@ -0,0 +1,355 @@
+      Medium.setState_phX(port_b.p,
+                          noEvent(actualStream(port_b.h_outflow)),
+                          noEvent(actualStream(port_b.Xi_outflow))) if
+          show_T "Medium properties in port_b";
+
+  Modelica.Fluid.Interfaces.FluidPort_a port_a(redeclare package Medium=Medium)
+    "Fluid connector a (positive design flow direction is from port_a to port_b)"
+    annotation (Placement(transformation(extent={{-110,-10},{-90,10}})));
+  Modelica.Fluid.Interfaces.FluidPort_b port_b(redeclare package Medium=Medium)
+    "Fluid connector b (positive design flow direction is from port_a to port_b)"
+    annotation (Placement(transformation(extent={{90,-10},{110,10}})));
+  Modelica.Blocks.Interfaces.RealInput on[num](
+    each final min=0,
+    each final max=1,
+    each final unit="1") "On signal for cooling towers"
+    annotation (Placement(transformation(extent={{-140,20},{-100,60}})));
+  Modelica.Blocks.Interfaces.RealInput TWetBul(
+    final unit="K",
+    displayUnit="degC")
+    "Entering air wetbulb temperature"
+    annotation (Placement(transformation(extent={{-140,-40},{-100,0}})));
+  Modelica.Blocks.Interfaces.RealOutput PFan[num](
+    each final quantity="Power",
+    each final unit="W")
+    "Electric power consumed by fan"
+    annotation (Placement(transformation(extent={{100,50},{120,70}})));
+  Modelica.Blocks.Interfaces.RealOutput TLvg[num](
+    each final unit="K",
+    each displayUnit="degC")
+    "Leaving water temperature"
+    annotation (Placement(transformation(extent={{100,20},{120,40}})));
+  Buildings.Applications.DHC.CentralPlants.Cooling.Subsystems.CoolingTowerParellel
+    cooTowSys(
+    final use_inputFilter=use_inputFilter,
+    redeclare final package Medium = Medium,
+    final num=num,
+    final show_T=show_T,
+    final m_flow_nominal=m_flow_nominal/num,
+    final dp_nominal=dp_nominal,
+    final ratWatAir_nominal=ratWatAir_nominal,
+    final TAirInWB_nominal=TAirInWB_nominal,
+    final TWatIn_nominal=TWatIn_nominal,
+    final dT_nominal=dT_nominal,
+    final PFan_nominal=PFan_nominal,
+    final energyDynamics=energyDynamics) "Cooling tower system"
+    annotation (Placement(transformation(extent={{-10,-10},{10,10}})));
+  Buildings.Fluid.Actuators.Valves.TwoWayEqualPercentage valByp(
+    redeclare final package Medium = Medium,
+    final m_flow_nominal=m_flow_nominal*0.0001,
+    final dpValve_nominal=dp_nominal,
+    final use_inputFilter=use_inputFilter) "Condenser water bypass valve"
+    annotation (Placement(transformation(
+        extent={{-10,-10},{10,10}},
+        origin={0,-40})));
+  Buildings.Fluid.Sensors.TemperatureTwoPort senTCWSup(
+    redeclare final package Medium = Medium,
+    final m_flow_nominal=m_flow_nominal,
+    final T_start=Medium.T_default)
+    annotation (Placement(transformation(extent={{60,10},{80,-10}})));
+  Modelica.Blocks.Sources.RealExpression TSetCWSup(
+    final y=max(TWetBul + dTApp, TMin))
```

Comment:
Avoid using expression, as it's not visually good as directly using graphical connection.